openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -774,8 +774,7 @@ Content-Length: 0
 
 The Credential Issuer provides a nonce value in the HTTP response with a 2xx status code and the following parameters included as top-level members in the message body of the HTTP response using the application/json media type:
 
-* `c_nonce`: REQUIRED. String containing a nonce to be used when creating a proof of possession of the key proof (see (#credential-request)).
-* `c_nonce_expires_in`: OPTIONAL. Number denoting the lifetime in seconds of the `c_nonce`. This value serves only as a hint to the Client, indicating how long the Credential Issuer is likely to accept the `c_nonce` as valid.
+* `c_nonce`: REQUIRED. String containing a nonce to be used when creating a proof of possession of the key proof (see (#credential-request)). This value MUST be unpredictable and unique for every response returned from the nonce endpoint.
```

Comment:
I'm not convinced we have consensus around "MUST be unique for every response returned from the nonce endpoint", particularly as there's no way to tie the request/response to a particular client etc. I'm sure people were talking about stateless implementations and given it's I believe an unprotected endpoint I think there are implementations that will want to limit the amount of work done in a call to the nonce endpoint.

Reply:
I don't think that requiring the nonce returned here to be unique precludes stateless implementations, for instance one could used signed timestamps if they want to achieve stateless nonces. However, I'm ok with dropping the uniqueness requirement if we believe some implementations for other reasons would want to say re-use a nonce over a short time period. I do believe the unpredictable language needs to be kept though that was inspired by the language in the DPoP RFC.

Reply:
Suggestion based on @jogu comment/

```suggestion
* `c_nonce`: REQUIRED. String containing a nonce to be used when creating a proof of possession of the key proof (see (#credential-request)). This value MUST be unpredictable.
```

Reply:
LGTM, thanks Tobias!